CHANGELOG.md

```diff
@@ -15,6 +15,11 @@ __BEGIN_UNRELEASED__
 ### Security
 __END_UNRELEASED__
 
+## [0.7.0] - 2019-08-14
+
+### Fixed
```

Comment:
This should also include the ref-forwarding fix in https://github.com/heap/react-native-heap/pull/124